src/components/Table/readme.md

```diff
@@ -695,3 +695,216 @@ const tableContainerStyles = { height: 300 };
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+`;
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+`;
+
+const StyledLink = styled.a.attrs(props => props.theme.rainbow)`
+    color: ${props => props.palette.brand.main}
+    &:hover {
+        color: ${props => props.palette.brand.main};
+    }
+    &:visited {
+        color: ${props => props.palette.brand.main};
+    }
+    &:active {
+        color: ${props => props.palette.brand.main};
+    }
+`;
+
+const ProfileLink = ({value, row}) => <StyledLink href={row.link}>{ value }</StyledLink>;
+
+function TableExample (props) {
+    const [ data, setData ] = useState(props.data);
+    const [ sortedBy, setSortedBy ] = useState(undefined);
+    const [ sortDirection, setSortDirection ] = useState('asc');
+    const [ selection, setSelection ] = useState([]);
+
+    function handleOnSort(event, field, nextSortDirection) {
+        let newData = [...data];
+
+        const key = x => x[field];
+        const reverse = nextSortDirection === 'asc' ? 1 : -1;
+
+        const sortedData = newData.sort((a, b) => {
+            a = key(a);
+            b = key(b);
+            return reverse * ((a > b) - (b > a));
+        });
+
+        setData(sortedData);
+        setSortedBy(field);
+        setSortDirection(nextSortDirection);
+    }
+
+    function handleOnDeleteElements() {
+        const newData = data.filter( e =>  !(selection.map(x=>x.id).includes(e.id)));
+        return setData(newData);
+    }
+
+    return (
+        <div className="rainbow-p-bottom_xx-large">
+            <GlobalHeader src="images/user/user3.jpg" />
+            <div className="rainbow-flex rainbow-justify_spread rainbow-align_center rainbow-p-around_large">
+                <div>
+                    <Title>
+                        Company Members
+                    </Title>
+                    <Subtitle>
+                        Total • {data.length}
+                    </Subtitle>
+                </div>
+                <ButtonGroup className="rainbow-m-right_medium">
+                    <ButtonIcon
+                        variant="border-filled"
+                        icon={<FontAwesomeIcon icon={faCog} />}
+                    />
+                    <ButtonMenu
+                        menuAlignment="right"
+                        menuSize="xx-small"
+                        icon={<FontAwesomeIcon icon={faEllipsisV} />}
+                    >
+                        <MenuItem label="Delete Selection" onClick={handleOnDeleteElements} />
+                    </ButtonMenu>
+                </ButtonGroup>
+            </div>
+            <Table
+                style={ tableContainerStyles }
+                keyField="id"
+                showCheckboxColumn
+                data={data}
+                onRowSelection={sel => setSelection(sel)}
```

Comment:
here you can simply use:
```onRowSelection={setSelection}```
since params passed to setSelection are the same onRowSelection receive